cmd/createorupdate/createorupdate.go

```diff
@@ -7,13 +7,14 @@ import (
 	"strings"
 
 	"github.com/ghodss/yaml"
-	"k8s.io/apimachinery/pkg/util/errors"
+	log "github.com/sirupsen/logrus"
 
 	"github.com/openshift/openshift-azure/pkg/api"
 	acsapi "github.com/openshift/openshift-azure/pkg/api"
 	"github.com/openshift/openshift-azure/pkg/api/v1"
 	"github.com/openshift/openshift-azure/pkg/plugin"
 	"github.com/openshift/openshift-azure/pkg/tls"
+	"k8s.io/apimachinery/pkg/util/errors"
```

Comment:
nit: move back

Reply:
yeah, saw that when looking at the changes again, already taken care of.  Dunno why my IDE keeps doing that... :angry: